model/client4.go

```diff
@@ -1721,6 +1721,16 @@ func (c *Client4) GetChannelStats(channelId string, etag string) (*ChannelStats,
 	}
 }
 
+// GetChannelMembersTimezones gets a list of timezones for a channel.
+func (c *Client4) GetChannelMembersTimezones(channelId string) ([]string, *Response) {
+	if r, err := c.DoApiGet(c.GetChannelRoute(channelId)+"/timezones", ""); err != nil {
```

Comment:
See above, but let's write the error handling here idiomatically as per other server-side changes going on.

Reply:
done. I can open a PR to fix the others in this files, if you agree @lieut-data